Cannot build locally

I cannot build locally since this PR was merged: https://github.com/SQLStreamStore/SQLStreamStore/pull/183

I get this error every single time I try:

```
/src/SqlStreamStore.Http.Tests/SqlStreamStore.Http.Tests.csproj : error NU1107: Version conflict detected for SqlStreamStore. Install/reference SqlStreamStore directly to project SqlStreamStore.Http.Tests to resolve this issue.  [/src/SqlStreamStore.sln]
/src/SqlStreamStore.Http.Tests/SqlStreamStore.Http.Tests.csproj : error NU1107:  SqlStreamStore.Http.Tests -> SqlStreamStore.TestUtils -> SqlStreamStore  [/src/SqlStreamStore.sln]
/src/SqlStreamStore.Http.Tests/SqlStreamStore.Http.Tests.csproj : error NU1107:  SqlStreamStore.Http.Tests -> SqlStreamStore.HAL 1.0.0-rc2-build00035 -> SqlStreamStore (>= 1.2.0-build00124). [/src/SqlStreamStore.sln]
```

--- a/src/SqlStreamStore.Http.Tests/SqlStreamStore.Http.Tests.csproj
+++ b/src/SqlStreamStore.Http.Tests/SqlStreamStore.Http.Tests.csproj
@@ -20,6 +20,7 @@
   <ItemGroup>
     <ProjectReference Include="..\SqlStreamStore.Http\SqlStreamStore.Http.csproj" />
     <ProjectReference Include="..\SqlStreamStore.TestUtils\SqlStreamStore.TestUtils.csproj" />
+    <ProjectReference Include="..\SqlStreamStore\SqlStreamStore.csproj" />
   </ItemGroup>
 
   <ItemGroup>

